src/Claim.test.ts

```diff
@@ -1,81 +1,195 @@
 /* tslint:disable:no-relative-imports */
+import * as jsig from 'jsonld-signatures'
 import { describe } from 'riteway'
-import { createClaim, isValidSignature, getClaimId, getClaimSignature, isValidClaim, canonizeClaim } from './Claim'
-import { Claim, ClaimType, ClaimAttributes, isClaim, Work } from './Interfaces'
+import { canonizeClaim, createClaim, getClaimId, isValidClaim, isValidSignature, signClaim } from './Claim'
+import { Claim, ClaimAttributes, ClaimType, Identity, isClaim, Work } from './Interfaces'
 
-const makeClaim = (attributes: ClaimAttributes) => {
-  const created = '2017-12-11T22:54:40.261Z'
-  const publicKey = '02db393ae2d566ceddd95a97fd88bc2897a0818528158261cec45087a58786f09d'
+// Generated:
+// const forge = require('node-forge')
+// const ed25519 = forge.pki.ed25519
+// const keypair = ed25519.generateKeyPair()
+// const bs58 = require('bs58')
+// bs58.encode(keypair.publicKey)
+const testPublicKeyEd25519Base58: string = 'JAi9YoyDdgBQLenyVzoXWH4C26wKMzHrjertxVrjLWTe'
+
+const testBadPublicKey: string = 'JAi9YoyDdgBQLenyVzoXWH4C26wKMzHrjertxVrjLWT5'
+
+// bs58.encode(keypair.privateKey)
+const testPrivateKeyEd25519Base58: string =
+  'LWgo1jraJrCB2QT64UVgRemepsNopBF3eJaYMPYVTxpEoFx7sSzCb1QysHeJkH2fnGFgHirgVR35Hz5A1PpXuH6'
+
+const testOwnerUrl = `data:,${testPublicKeyEd25519Base58}`
+
+export const TestPublicKeyUrl = `data:,${testPublicKeyEd25519Base58}`
```

Comment:
Can't find a reason why this is exported? by accident?

Reply:
Fixed